v5/heroku.go

```diff
@@ -8,21 +8,20 @@
 // The Service struct has all the methods you need
 // to interact with heroku API.
 //
-package v5
+package heroku
```

Comment:
I think `generate` script might need some work. This was auto changed by the script.